spec/unit/lib/temporal/testing/local_workflow_context.rb

```diff
@@ -0,0 +1,110 @@
+require 'temporal/testing'
+require 'temporal/workflow'
+require 'temporal/api/errordetails/v1/message_pb'
+
+describe Temporal::Testing::LocalWorkflowContext do
+  let(:namespace) { 'default-namespace' }
+  let(:task_queue) { 'default-task-queue' }
+  let(:workflow_id) { 'workflow_id_1' }
+  let(:run_id) { 'run_id_1' }
+  let(:execution) { Temporal::Testing::WorkflowExecution.new }
+  let(:workflow_context) do
+    Temporal::Testing::LocalWorkflowContext.new(
+      execution,
+      workflow_id,
+      run_id,
+      []
+    )
+  end
+  let(:async_token) do
+    Temporal::Activity::AsyncToken.encode(
+      namespace,
+      1, # activity ID starts at 1 for each workflow
+      workflow_id,
+      run_id
+    )
+  end
+
+  class TestFailedActivity < Temporal::Activity
+    namespace namespace
```

Comment:
I'm still puzzled why this works… ^

Reply:
Honestly, I didn't expect this to work either but it does. I can rename the variable to something like `test_namespace` to clarify

Reply:
This isn't really necessary. I can just load the default namespace from the configuration which simplifies this quite a bit.

Reply:
Perfect 👍